templates/master/00-master/baremetal/files/baremetal-keepalived-keepalived.yaml

```diff
@@ -65,6 +81,11 @@ contents:
         virtual_router_id {{`{{ .Cluster.IngressVirtualRouterID }}`}}
         priority 40
         advert_int 1
+        unicast_peer {
```

Comment:
How do we handle the case where router pods run in worker node?

Reply:
Are you talking about the ingress VIP?

Reply:
Yep

Reply:
Ok. Addressed.